tests/chip_id_tests.py

```diff
@@ -10,7 +10,7 @@
 import binascii
 
 import logging
-logging.basicConfig(level=logging.INFO, format="%(message)s")
+#logging.basicConfig(level=logging.INFO, format="%(message)s")
```

Comment:
should this have gone with the logging PR instead? (fine, just weird that it's here)